src/operators/rbl.cc

```diff
@@ -211,7 +211,7 @@ bool Rbl::evaluate(Transaction *t, RuleWithActions *rule,
         return false;
     }
 
-    rc = getaddrinfo(host.c_str(), NULL, NULL, &info);
```

Comment:
Same here, can we elaborate on this change? given that `if (rc != 0) {` is always false.